meta-mender-variscite/recipes-multimedia/alsa/imx-alsa-plugins_git.bbappend

```diff
@@ -0,0 +1 @@
+SRC_URI = "git://github.com/nxp-imx/imx-alsa-plugins.git;protocol=https;branch=${SRCBRANCH}"
```

Comment:
see above, misses machine override

Reply:
Added in commit b5550c9e002d37f851e9145d6b6a3e3a479855dc

Reply:
Added in commit b5550c9e002d37f851e9145d6b6a3e3a479855dc